sdk/core/azure-core/azure/core/_match_conditions.py

```diff
@@ -28,9 +28,10 @@
 
 
 class MatchConditions(Enum):
-    """An enum to describe match conditions. """
+    """An enum to describe match conditions."""
+
     Unconditionally = 1  # Matches any condition
     IfNotModified = 2  # If the target object is not modified. Usually it maps to etag=<specific etag>
     IfModified = 3  # Only if the target object is modified. Usually it maps to etag!=<specific etag>
-    IfPresent = 4   # If the target object exists. Usually it maps to etag='*'
-    IfMissing = 5   # If the target object does not exist. Usually it maps to etag!='*'
+    IfPresent = 4  # If the target object exists. Usually it maps to etag='*'
```

Comment:
I know this is black only, but why do we have a numerical enum for this? And the comments are in the wrong format - we want them to show in intelligence, [docs](https://learn.microsoft.com/en-us/python/api/azure-core/azure.core.matchconditions?view=azure-python) etc.

Reply:
Fixed the comments:

![image](https://user-images.githubusercontent.com/1050156/210153864-48166733-d2ce-400d-9f32-0556b07c71e9.png)